### PR TITLE
allow to manage tokens without governance duration

### PR DESCRIPTION
**Pull request by johannbarbie**
resolves https://github.com/leapdao/leap-contracts/issues/218

### CODE REVIEW

File: contracts/MinGov.sol [hunk 1]
```diff
@@ -106,4 +106,20 @@ contract MinGov is Ownable {
     return bytes4(_msgData[3]) >> 24 | bytes4(_msgData[2]) >> 16 | bytes4(_msgData[1]) >> 8 | bytes4(_msgData[0]);
   }
 
+  // proxy function to manage tokens without governance delay
+  // token types: 0 = ERC20, 1 = ERC721, 2 = ERC1948
+  function registerToken(address payable _subject, address _token, uint256 _type) public onlyOwner {
```
Comment (troggy) [inline contracts/MinGov.sol]: not related, but can we make `Vault.registerToken` to work the same (accept `_type` argument) and drop `registerNST` whatsoever? 🤔

this function becomes easier as well

**Comment by troggy** [comment]
@johannbarbie merging or you want to change something else?

**Comment by johannbarbie** [comment]
i can clean up the technical debt. but is updating the main-net part of this bounty, or not? depending on that i would fork it out into a separate bounty, or not.

**Comment by troggy** [comment]
updating mainnet is not the part of this bounty (should be, but I overlooked this). Changing the Vault contract is also totally out of scope here and should be done separately.

**Comment by troggy** [comment]
however, if change the Vault later we will need to change MinGov again, so it makes sense to do this in one go. Should I increase the bounty size?

**Comment by johannbarbie** [comment]
ok, then let's change both in 1 go. i'll update vault contract. and then we do testnet and mainnet upgrade / deployment in a separate bounty.

**Comment by johannbarbie** [comment]
@troggy want to review: https://github.com/leapdao/leap-contracts/pull/222 ?

**Comment by johannbarbie** [comment]
deployment bounty here: https://github.com/leapdao/leap-contracts/issues/223

**Comment by johannbarbie** [comment]
superseded by https://github.com/leapdao/leap-contracts/pull/222

**Pull request closed without merging**